docs/concepts/whatisvpp/modularflexibleextensible.rst

```diff
@@ -0,0 +1,43 @@
+.. _modularflexibleextensible:
+
+.. toctree::
+
+Modular, Flexible, and Extensible
+***********************************
+
+The FD.io VPP platform is built on a ‘packet processing graph’. This modular
+approach means that anyone can ‘plugin’ new graph nodes. This makes
+extensibility rather simple, and it means that plugins can be customized
```

Comment:
This makes FD.io VPP easily extensible, and it means that ....

Reply:
FD.io is also configurable through it's Low-Level API.

Reply:
Should we make a note to include a link to this Low-level API docs once that section's done?